samples/DataSourceWithProcessingAndExtensions/Sample/CustomDataSource.cs

```diff
@@ -0,0 +1,62 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using Microsoft.Performance.SDK.Processing;
+
+namespace DataExtensionsSample
+{
+    /// <summary>
+    /// This custom data source defines the type of data it will process. In this case, a file with a "des" filename
+    /// extension. 
+    /// </summary>
+    [CustomDataSource(
+        "{96630F86-3FEA-4D21-A72E-8449D8784272}",
```

Comment:
Can we add a comment indicating that GUID should be unique to your custom data source? That makes it clear that this GUID can be any GUID.